pages/index.tsx

```diff
@@ -4,36 +4,34 @@ import Link from 'next/link';
 import Container from '@styles/Container.styles';
 
 // Graphql
-import { ssrExchange, dedupExchange, cacheExchange, fetchExchange, useQuery } from 'urql';
-import { NAVIGATION_MENU_QUERY } from '../src/graphql/queries';
-import { withUrqlClient, initUrqlClient } from 'next-urql';
+import { NAVIGATION_MENU_QUERY } from '../graphql/queries';
+import { GraphQLClient } from 'graphql-request';
 
 const BASE_URL = 'https://dotcms.com/api/v1/graphql';
 
-export function Home(): JSX.Element {
-    const [res] = useQuery({ query: NAVIGATION_MENU_QUERY });
-    const { data } = res;
+export default function Home({ data }: DotcmsDocumentationCollection): JSX.Element {
     return (
         <Container>
             <Head>
                 <title>Documentation</title>
                 <link href="/favicon.ico" rel="icon" />
             </Head>
+            <h1>Hey</h1>
             <nav>
                 <DotCollection data={data.DotcmsDocumentationCollection[0]} />
             </nav>
         </Container>
     );
 }
 
-const DotCollection = ({ data }: any) => {
+const DotCollection = ({ data }: DotcmsDocumentationData) => {
     if (!data.dotcmsdocumentationchildren) {
         return null;
     }
 
     return (
         <ul>
-            {data.dotcmsdocumentationchildren.map((item) => (
+            {data.dotcmsdocumentationchildren.map((item: DotcmsDocumentation) => (
```

Comment:
```suggestion
            {data.map((item: DotcmsDocumentation) => (
```

Reply:
Why did you change 'data.dotcmsdocumentationchildren' to 'data'? data.dotcmsdocumetnationchildren is the array we are iterating.